### PR TITLE
Harshpavuluri/plate pcb compatibility change

### PR DESCRIPTION
**Pull request by harshpavuluri**
Updated FAQ questions to include recent pins from #tech-and-build-support.

1. Updated "Will plate fit from different designs?"
2. Created "Will X PCB fit into a Keycult keyboard?"

### CODE REVIEW

File: pages/mydoc/faq_keycult_faq.md [hunk 1]
```diff
@@ -68,7 +68,11 @@ This is meant to be a collection of commonly asked questions and their answers.
                         </div>
                         <div id="collapseFour" class="panel-collapse collapse">
                             <div class="panel-body">
-                                No the plates are not compatible.
+                                <ul>
+                                    <li>No. 1 vs. No. 2 plates are not interchangeable</li>
+                                    <li>No. 1 Rev. 2 plates will work on No. 1 Rev. 1 (without alignment pins)</li>
+                                    <li>No. 2 Rev. 1 plates will work on original No. 2</li>
```
Comment (swangful) [inline pages/mydoc/faq_keycult_faq.md:74]: while were in here, can you please add here `(without alignment pins)`

File: pages/mydoc/faq_keycult_faq.md [hunk 2]
```diff
@@ -68,7 +68,11 @@ This is meant to be a collection of commonly asked questions and their answers.
                         </div>
                         <div id="collapseFour" class="panel-collapse collapse">
                             <div class="panel-body">
-                                No the plates are not compatible.
+                                <ul>
+                                    <li>No. 1 vs. No. 2 plates are not interchangeable</li>
+                                    <li>No. 1 Rev. 2 plates will work on No. 1 Rev. 1 (without alignment pins)</li>
```
Comment (swangful) [inline pages/mydoc/faq_keycult_faq.md:73]: lets note here that they wont work on OG No. 1

File: pages/mydoc/faq_keycult_faq.md (hunk 1 above)
Comment (harshpavuluri) [reply pages/mydoc/faq_keycult_faq.md:74]: Is it for the No. 2 bullet point?

File: pages/mydoc/faq_keycult_faq.md [hunk 3]
```diff
@@ -99,6 +103,24 @@ This is meant to be a collection of commonly asked questions and their answers.
...
+                                <a class="noCrossRef accordion-toggle" data-toggle="collapse" href="#collapseEightteen">Will X PCB fit a Keycult keyboard?</a>
+                            </h4>
+                        </div>
+                        <div id="collapseEightteen" class="panel-collapse collapse noCrossRef">
+                            <div class="panel-body">
+                                The recommended supported PCBs are the ones sold on the Keycult store at <a href="https://keycult.com/collections/accessories" target="_blank">here</a>. The following PCBs will also fit as well.
+                                <ul>
+                                    <li>No. 1/60: WT60-G</li>
+                                    <li>No. 1/65 or No. 2/65: WT65-G</li>
+                                    <li>TKL: WT80-G (Note: This PCB will NOT work on the original No. 1)</li>
```
Comment (swangful) [inline pages/mydoc/faq_keycult_faq.md:118]: we can say for OG No. 1 they can refer to this [list](https://docs.google.com/spreadsheets/d/1GpM--fKeDw_tIDZj4m2jmbMavSyXeOu3QHzfTJwz-IM/edit#gid=0)

File: pages/mydoc/faq_keycult_faq.md (hunk 1 above)
Comment (swangful) [reply pages/mydoc/faq_keycult_faq.md:74]: yeppers!

**Comment by harshpavuluri** [comment]
Just pushed changes according to comments and added sub-bullet points so "notes" are clear

**Review by swangful — APPROVED** [review]

**Pull request merged**